dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -259,12 +259,12 @@ BEGIN
     LTEXT "&Domeniul:", 1110, 9, 99, 91, 8
     EDITTEXT 1102, 102, 95, 154, 14, ES_UPPERCASE | ES_AUTOHSCROLL
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN, 9, 116, 247, 1
-    AUTOCHECKBOX "Păstrea&ză acest nume și parolă pentru acești utilizatori:", 1101, 12, 123, 243, 13, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
+    AUTOCHECKBOX "Sal&vare a acestor nume și parolă pentru acești utilizatori:", 1101, 12, 123, 243, 13, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
     AUTORADIOBUTTON "Pentru mine, e&xclusiv", 1622, 25, 137, 229, 11, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
     AUTORADIOBUTTON "Pentru &oricine din acest calculator", 1623, 25, 151, 230, 11, BS_LEFT | BS_TOP | BS_MULTILINE
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 9, 167, 247, 1
     DEFPUSHBUTTON "&Apelează", 1590, 8, 173, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 1591, 71, 173, 60, 14
+    PUSHBUTTON "&Revocare", 1591, 71, 173, 60, 14
```

Comment:
This change introduces an accelerator collision with the line below
 PUSHBUTTON "P&roprietăți", 1107, 133, 173, 60, 14
Please check your romanian Windows to find out which of both lines need to be changed.